### DISCUSSION TITLE
Change the method of replacing placeholders in the message.

### DISCUSSION THREAD

**Discussion by ikuno9233** [laravel/framework#48734] [Ideas]
Hello.
I'm a Japanese engineer.
I'm not good at English, but like you, I want to make Laravel better.

#### Proposal
Change the method of replacing placeholders in the message.

#### Background
I created a Japanese language file using `laravel-lang/lang v13.2.5`, but I noticed that the `:Other` placeholder was not replaced when a validation error message was displayed.
However, `:attribute` and `:Attribute` have been replaced.
When I investigated the source code, I found that `:other` must be all lowercase letters to be replaced.

`:attribute` has different results depending on the case of the placeholder.
The same goes for placeholder replacement using the `trans()` helper.

```
Example: :attribute -> field name, :Attirubte -> Field name, :ATTRIBUTE -> FIELD NAME
```

This is currently an issue with the `laravel-lang/lang` language file.
However, `:other` is a placeholder that can be replaced with the attribute name, just like `:attribute`.

I think it would lead to confusion if these replacement methods are different, so I would like to suggest changing the replacement method so that it is the same as the `:attribute` replacement method.

Also, to maintain consistency, I think `:min`, `:date`, etc. should be replaced with `:Min`, `:DATE` as well. (Although the replacement result remains the same)
However, this change may be redundant since `:min`, `:max`, etc. are replaced with numerical values.

Please let me know your opinions on my thoughts.

**Comment by henzeb** [comment]
I agree. It does not make sense  to have uppercase and lowercase placeholders. The word means the same...

**Reply by ikuno9233** [reply]
Thank you for reply!

I'm sorry if I'm wrong, but I think you've misunderstood my intent.
I was going to change it as below.

```
current
:attribute -> field name, :Attirubte -> Field name, :ATTRIBUTE -> FIELD NAME
:other -> other field name, :Other -> (not replaced), :OTHER -> (not replaced)

change
:attribute -> field name, :Attirubte -> Field name, :ATTRIBUTE -> FIELD NAME
:other -> other field name, :Other -> Other field name, :OTHER -> OTHER FIELD NAME
```

The essence of the problem is not whether uppercase or lowercase letters are converted or not, but the `:attribute` and `:other` placeholders are handled differently.
It might be a good idea to make them all lowercase!

```
change
:attribute -> field name, :Attribute -> (not replaced), :ATTRIBUTE -> (not replaced)
:other -> other field name, :Other -> (not replaced), :OTHER -> (not replaced)
```
